### PR TITLE
Add steps to make a user admin

### PR DESCRIPTION
**Pull request by keianhzo**
### What
Add steps to the README.md to connect to the admin panel after running the stack.

### Why
Connecting to the admin panel is a common thing that developers might need to use when updating either the admin or the client.

Fixes https://github.com/mozilla/hubs-compose/issues/8

### CODE REVIEW

**Review by johnshaughnessy — COMMENTED** [review]
It may be easier to run an interactive elixir shell in reticulum's docker container:
```bash
docker compose exec -it reticulum iex -S mix
```
and update the account with `Ecto` functions:
```elixir
Ret.Account.find_or_create_account_for_email("jshaughnessy@mozilla.com")
|> Ecto.Changeset.change( %{ is_admin: true } )
|> Ret.Repo.update()
```

**Comment by bryanenders** [comment]
> ```shell
> docker compose exec -it reticulum iex -S mix
> ```

Or better yet `bin/iex -S mix`

**Comment by keianhzo** [comment]
I totally missed that this was already in the Ret docs. Makes sense to just link to that:

> I was hesitant to [duplicate instructions](https://github.com/mozilla/reticulum/blob/342eb9e5b8c8648d0e2b2025c0089d576d0e1924/README.md#6-creating-an-admin-user) from the individual service READMEs since they could become out of sync. What if we just add a note to the README to follow the steps for each service?

File: README.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ prompts:
 * [8080: Hubs Client](https://hubs.local:8080)
 * [4000: Reticulum](https://hubs.local:4000)
 
+### Admin panel access
+
+To connect to the admin panel you will need to manually promote an account to admin::
```
Comment (bryanenders) [inline README.md:50]:
```suggestion
To connect to the admin panel you will need to manually promote an account to admin:
```

**Review by bryanenders — APPROVED** [review]

**Pull request merged**